scene/gui/control.cpp

```diff
@@ -795,9 +814,10 @@ void Control::_set_layout_mode(LayoutMode p_mode) {
 	bool list_changed = false;
 
 	if (p_mode == LayoutMode::LAYOUT_MODE_POSITION || p_mode == LayoutMode::LAYOUT_MODE_ANCHORS) {
-		if ((int)get_meta("_edit_layout_mode", p_mode) != (int)p_mode) {
-			list_changed = true;
+		if ((int)get_meta("_edit_layout_mode", p_mode) == (int)p_mode) {
+			return;
 		}
+		list_changed = true;
```

Comment:
This will be irrelevant after you rebase, but please don't change any logic in this PR. This is not a valid change, and this is not in the scope of this PR.